test/index_spec.js

```diff
@@ -131,6 +131,29 @@ describe('textPosition', () => {
       ].join('')
       assert.equal(text, expected)
     })
+
+    it('can describe a range starting at an empty element', () => {
```

Comment:
These are a couple of extra tests suggested both by the coverage report and the anchoring tests in the Hypothesis client.